administration/src/mui-modules/application/util/FormType.ts

```diff
@@ -2,7 +2,12 @@ import { ReactElement } from 'react'
 
```

Comment:
Can you explain these changes?

Reply:
> The any type in TypeScript is a dangerous "escape hatch" from the type system. Using any disables many type checking rules and is generally best used only as a last resort or when prototyping code. This rule reports on explicit uses of the any keyword as a type annotation.

See https://typescript-eslint.io/rules/no-empty-object-type. Basically, using an empty object `{}` as type represents any non-nullish value, including literals like 0 and "". Since in this case we only want actual objects (or more specifically, records), I constrained the type to only records. Since we don't know the types of the values, th  type is `Record<string, unknown>`. Does that make sense?

Reply:
Thanks for the explanation! (I guess one could argue whether `Options` has to be an actual object, but it's totally fine to restrict it to actual objects).